lib/tokens.ts

```diff
@@ -171,94 +139,83 @@ export type ThemeV3WithDarkModeSupportType = {
 }
 export type AllThemesType = ThemeV3Type & ThemeV3WithDarkModeSupportType
 
+// These are for styled-system
+export type SpacingUnitsTheme = { space: Record<SpacingUnit, any> }
+export type ColorsTheme = { colors: Record<Color, any> }
+
+const THEME = {
```

Comment:
```suggestion
const v3 = {
```
why not just name this with the key directly?